### PR TITLE
added build instructions in the readme

### PR DESCRIPTION
**Pull request by jakehemmerle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ### x25519-chacha20poly1305
 
+> NOTE: make sure you have your C archiver and compiler env vars set, or `secp256k1-sys` will fail to build
+>
+> e.g.
+> `AR=/opt/homebrew/opt/llvm/bin/llvm-ar`
```
Comment (benschac) [inline README.md:6]: small nit: 
be a little more explicit
```
export AR=/opt/homebrew/opt/llvm/bin/llvm-ar
export CC=/opt/homebrew/opt/llvm/bin/clang make
```
also, where vars should be added. zshrc, bashrc, or whatever environment the developer uses.

Comment (benschac) [reply README.md:6]: also, my CC is different from this one. Does having `make` appended to `CC` matter?

**Review by benschac — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jakehemmerle) [reply README.md:6]: Those are just the environmental variables that need to be set. each shell is going to be different, zsh (and i think bash) does it via exports.If you want to set them them in rc then it makes sense

Comment (jakehemmerle) [reply README.md:6]: You do not want to have make in your CC; make is the program that you're running (it executes the Makefile in the root directory), and it uses the CC env var while it's building the system.

**Review by ok-john — DISMISSED** [review]

**Review by ok-john — APPROVED** [review]

**Review by ok-john — APPROVED** [review]

**Pull request merged**